packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -47,9 +52,15 @@ function getKey(node) {
 
 // NOTE: This function can be easily bound as .bind(file, classRef, superRef)
 //       to make it easier to use it in a loop.
-function extractElementDescriptor(/* this: File, */ classRef, superRef, path) {
+function extractElementDescriptor(
+  /* this: File, */
```

Comment:
With TS we can actually specify the `this` type annotation. I _think_ this refers to `import type { File } from "@babel/core"`.